_episodes/03-using-packages-and-channels.md

```diff
@@ -38,7 +38,7 @@ For a more details of the conda package specification, including discussions of
 metadata files, see the [docs](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html).
 
 As an example of Conda package structure consider the [Conda](https://pytorch.org/) package for 
-Python 3.6 version of `PyTorch` targeting a 64-bit Mac OS, `pytorch-1.1.0-py3.6_0.tar.bz2`.
+Python 3.10 version of `PyTorch` targeting a 64-bit Mac OS, `pytorch-1.13.1-py3.10_cuda11.7_cudnn8.5.0_0.tar.bz2`.

```

Comment:
Does this package still bundle `caffe2`?

Reply:
I don't think the one from PyTorch do but can't see it stated anywhere but it looks like [caffe2](https://pytorch.org/tutorials/advanced/super_resolution_with_caffe2.html) make Conda packages of PyTorch that include it.

Not sure though, not something I know anything about.